packages/safety-suite-api/src/permissions.ts

```diff
@@ -184,6 +185,38 @@ export class UserWithPermissions extends UserWrapper {
     return [];
   }
 
+  /**
+   * Gets array of indirectly accessible groups by template and action.
+   * @param idOrAlias ID or alias of the model template.
+   * @param action Permitted action type.
+   */
+  getIndirectGroupsByModelTemplate(
+    idOrAlias: Alias | Id,
+    action: ModulePermissionActions
+  ): ModelGroup[] {
+    const template = this.modelTemplates.find(
+      ({alias, id}) => idOrAlias === alias || idOrAlias === id
+    );
+    if (!template) {
+      return [];
+    }
+    if (this.admin) {
+      return this.modelGroups;
+    }
+    if (this.isLegacy()) {
+      return [];
```

Comment:
Do we want this to not be backwards compatible? If RPH is disabled then this will always return empty for non-admin users? I think the alternative is just returning all like the `getGroupsByModule` method works. Either way is ok but its an important distinction.

Reply:
Since there is no such concept like indirect access in old permissions model I assume that this will return just empty array.
This function is used coupled with `getGroupsByModule`, so frontend explicitly check for direct (which has backward compatibility) and indirect access.